src/assets/data/faq/index.js

```diff
@@ -325,24 +325,22 @@ const data = {
     },
 
     {
-      question: "How to Meshery Extension Points work?",
+      question: "How do Meshery Extension Points work?",
       category: "Meshery Integrations",
+      link: 'https://docs.meshery.io/extensibility',
+      linktext: 'Meshery Extensibility',
       answer: [
-        "Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.",
+        "Meshery is an extensible platform. Customize Meshery using any of its purpose-built extension points.",
+        'Take advantage of these extension points for working with different service meshes via adapters, different load generators and different providers. Use Meshery’s REST API or its GraphQL API both of which allow you to connect to any Kubernetes-native app.'
       ],
     },
     {
       question: "How can I request a new integration?",
       category: "Meshery Integrations",
+      link: 'https://layer5.io/company/contact',
+      linktext: 'Contact Us',
       answer: [
-        "Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.",
-      ],
-    },
-    {
-      question: "Does Meshery manage service meshes?",
```

Comment:
This question should be there as well.
The answer for the same is already provided by Debopriya in the issue comments, update accordingly.